src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -445,8 +448,30 @@ public int GetIndexesCount()
 
             return count;
         }
+
+        internal SupportedFeatureTypes GetSupportedFeatures()
+        {
+            var supportedFeatures = new SupportedFeatureTypes();
+            if (SupportedFeatures.Contains(nameof(SupportedFeatureTypes.ThrowRevisionKeyTooBigFix)))
```

Comment:
I agree

Reply:
do not understand what we are trying to do here. IMO we could have a get-only property for supported features and on first access it would build that?

Reply:
done